packages/auth/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Auth",
+  "version": "1.0.0",
+  "main": "index.js",
```

Comment:
i assume this is just boilerplate but it might actually be a good idea to make an index file that just re-exports everything so that you can just `import {AuthWall} from "@recidiviz/auth"` or whatever